taxcalc/records_variables.json

```diff
@@ -720,8 +720,8 @@
     "iitax": {
       "type": "float",
       "desc": "Total federal individual income tax liability; appears as INCTAX variable in tc CLI minimal output",
-      "form": {"2013-2013": "1040 line 61 minus line 56 minus line 60a",
-               "2014-2016": "1040 line 63 minus line 57 minus line 62a"}
+      "form": {"2013-2013": "1040 line 61 minus lines 56, 60a, 64a, 65, and 66",
```

Comment:
@donboyd5 How do you feel about this update to the documentation of the computation of `iitax` from form 1040?

Reply:
@donboyd5 Let me know if you think these updates are satisfactory.  Thanks!

Reply:
@jdebacker thanks for this, it looks great, and sorry for the slow response.